test/integration_tests/test_select_executor.py

```diff
@@ -88,21 +88,59 @@ def test_should_load_and_select_in_table(self):
         actual_batch.sort()
         expected_batch = list(create_dummy_batches())
         self.assertEqual([actual_batch], expected_batch)
+    
+    def test_should_select_star_in_table(self):
+        select_query = "SELECT * FROM MyVideo;"
+        actual_batch = execute_query_fetch_all(select_query)
+        actual_batch.sort()
+        expected_batch = list(create_dummy_batches())[0]
+        self.assertEqual(actual_batch, expected_batch)
+
+        select_query = "SELECT * FROM MyVideo WHERE id = 5;"
+        actual_batch = execute_query_fetch_all(select_query)
+        expected_batch = list(create_dummy_batches(filters=[5]))[0]
+        self.assertEqual(actual_batch, expected_batch)
+
+    def test_should_select_star_in_nested_query(self):
+        select_query = """SELECT * FROM (SELECT * FROM MyVideo) AS T;"""
+        actual_batch = execute_query_fetch_all(select_query)
+        actual_batch.sort()
+        expected_batch = list(create_dummy_batches())[0]
+        expected_batch.modify_column_alias('T')
+        self.assertEqual(actual_batch, expected_batch)
+
+        select_query = """SELECT * FROM (SELECT id FROM MyVideo) AS T;"""
+        actual_batch = execute_query_fetch_all(select_query)
+        actual_batch.sort()
+        expected_rows = [{"T.id": i} for i in range(NUM_FRAMES)]
+        expected_batch = Batch(frames=pd.DataFrame(expected_rows))
+        self.assertEqual(actual_batch, expected_batch)
+
+    @unittest.skip('Not supported in current version')
+    def test_select_star_in_lateral_join(self):
```

Comment:
Shouldn't it return all the columns (including columns from UDF)? SO basically 4 columns?

Reply:
The column binding for function or lateral join is not correct. The assert is a placeholder. But the query does not run. So I am not able to figure out the actual output column names.